### PR TITLE
revert: rollback changes from #2171 and #2181

### PR DESCRIPTION
**Pull request by rpenido**
## Description
We refactored the way we store content tag data while exporting the XBlock and some previous changes are not needed anymore. This reverts the following PRs:
- #2171
- #2181

## More information
- Related to https://github.com/openedx/modular-learning/issues/180
- Discussion here: https://github.com/openedx/edx-platform/pull/34270#discussion_r1508474276

Depends on:
- https://github.com/openedx/openedx-learning/pull/164
- https://github.com/openedx/edx-ora2/pull/2181

## Testing instructions
- Make sure the tests cover the various cases.

___
Private ref: [FAL-3621](https://tasks.opencraft.com/browse/FAL-3621)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @rpenido! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: openassessment/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Initialization Information for Open Assessment Module
 """
 
-__version__ = '6.2.0'
+__version__ = '6.3.0'
```
Comment (pomegranited) [inline openassessment/__init__.py:5]: nit: please merge latest master and update this version.

**Review by pomegranited — APPROVED** [review]
:+1: Thank you for reverting this change @rpenido !

Could you please resolve the conflicts and bump the version again?

- [x] I tested this while testing https://github.com/openedx/edx-platform/pull/34270
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A
- [x] <strike>User-facing strings are extracted for translation</strike> N/A

**Comment by rpenido** [comment]
This is ready for review/merge @pomegranited!

**Comment by pomegranited** [comment]
I re-tested https://github.com/openedx/edx-platform/pull/34270 with this revert, and it's all working fine.

**Comment by openedx-webhooks** [comment]
@rpenido 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-ora2&entry.867055334=https://github.com/openedx/edx-ora2/pull/2184&entry.1484655318=https://github.com/rpenido&entry.752974735=2024-03-01+14:20&entry.1917517419=2024-03-07+00:28&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**